getting-started-github-graphql-api.md

```diff
@@ -40,59 +42,59 @@ When interacting with GitHub's GraphQL API, it would be great to fill your own a
 
 ### Exercises:
 
-* read more about the different terms in {{% a_blank "GitHub's glossary" "https://help.github.com/articles/github-glossary/" %}}
-  * what is a GitHub organization and GitHub user?
-  * what are repositories, issues and pull requests?
-  * what are GitHub repository stars and GitHub repository watchers?
-* create enough repositories in order to use the pagination feature later
-* create pull requests and issues in a few of your GitHub repositories
+* Read more about the different terms in {{% a_blank "GitHub's glossary" "https://help.github.com/articles/github-glossary/" %}}. Consider these questions:
+  * What is a GitHub organization and GitHub user?
+  * What are repositories, issues and pull requests?
+  * What are GitHub repository stars and GitHub repository watchers?
+* Create or fork enough epositories to use the pagination feature.
+* Create pull requests and issues in a few of your GitHub repositories.
 
 {{% chapter_header "Read/Write Data with GitHub's Personal Access Token" "github-personal-access-token" %}}
 
-In order to use GitHub's GraphQL API, you need to generate a personal access token on their website. The access token makes sure that you can be authorized as a user who interacts with their data. Basically, it allows you to read and write data in your name. You can {{% a_blank "follow their step by step instructions" "https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line" %}} to obtain the personal access token. While getting your token, make sure to check all the necessary scopes (permissions) for it. You will need sufficient read and write permissions in order to implement a well-rounded GitHub client later.
+To use GitHub's GraphQL API, you need to generate a personal access token on their website. The access token autorizes users to interact with data, to read and write it under your username. {{% a_blank "Follow their step by step instructions" "https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line" %}} to obtain the personal access token, and be sure to check the necessary scopes (permissions) for it, as you will need them to implement a well-rounded GitHub client later.
 
 {{% pin_it_image "react graphql github" "img/posts/getting-started-github-graphql-api/github-personal-access-token.jpg" "is-src-set" %}}
 
-Later, the personal access token can be used to interact with GitHub's GraphQL API. Basically it is your private key to access (read) and manipulate (write) GitHub's data, so you shouldn't share it with any third-parties.
+Later, the personal access token can be used to interact with GitHub's GraphQL API.  Be careful not to share these authorizations it with any third parties.
```

Comment:
"not to share these authorizations it with any third parties" to "not to share these authorizations with any third parties"